src/test/java/com/conveyal/gtfs/GTFSTest.java

```diff
@@ -489,6 +489,31 @@ public void canLoadFeedWithLongFieldValues () {
         );
     }
 
+    /**
+     * Tests that a GTFS feed with a service id without calendar days assigned
```

Comment:
"calendar days" sounds to me like "dates". A better phrase would be "days of the week."